sycl/include/sycl/ext/intel/esimd/memory.hpp

```diff
@@ -5917,32 +5933,17 @@ atomic_update(AccessorTy acc, Toffset byte_offset, simd_mask<N> mask) {
 ///               simd<T, N> src0, props = {});                  // (acc-au1-2)
 ///
 /// simd<T, N>
-/// atomic_update(AccessorT acc, simd<Toffset, N> byte_offset,
-///               simd_view<T, RegionTy> src0,
-///               simd_mask<N> mask, props = {});                // (acc-au1-3)
-/// simd<T, N>
-/// atomic_update(AccessorT acc, simd<Toffset, N> byte_offset,
-///               simd_view<T, RegionTy> src0,
-///               props = {});                                   // (acc-au1-4)
-///
-/// simd<T, N>
-/// atomic_update(AccessorT acc, simd_view<Toffset, OffsetRegionTy> byte_offset,
+/// atomic_update(AccessorT acc, simd_view<OffsetObjT, OffsetRegionTy>
+/// byte_offset,
 ///               simd<T, N> src0,
-///               simd_mask<N> mask, props = {});                // (acc-au1-5)
+///               simd_mask<N> mask, props = {});                // (acc-au1-3)
```

Comment:
nit: looks like `clang-format` changed this oddly. It would look better if `simd_view` was on the next line together with `byte_offset`.